templates/account.html

```diff
@@ -43,22 +43,32 @@ <h3>My snaps</h3>
                   {% else %}
                     {% for snap in user_snaps|sort %}
                         <tr>
-                            <td class="u-vertically-center">
-                                {% if user_snaps[snap].icon_url %}
-                                    <img src="{{ user_snaps[snap].icon_url }}" width="32" height="32" />&nbsp;
-                                {% else %}
-                                    <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" width="32" height="32" />&nbsp;
-                                {% endif %}
-                                {% if user_snaps[snap].uploaded %}
-                                    <a href="/account/snaps/{{snap}}/market" class="u-no-margin--top">{{snap}}</a>
+                            <td class="u-vertically-center p-snap-list">
+                              {% if user_snaps[snap].uploaded %}
+                                <a href="/account/snaps/{{snap}}/market" class="u-no-margin--top u-vertically-center">
+                              {% endif %}
+                              {% if user_snaps[snap].icon_url %}
+                                  <img src="{{ user_snaps[snap].icon_url }}" width="32" height="32" class="p-snap-list__image" />
```

Comment:
You added a scss file, maybe those lines ` width="32" height="32"` could be added in it

Reply:
done